queue_services/entity-digital-credentials/src/entity_digital_credentials/config.py

```diff
@@ -0,0 +1,166 @@
+
+All items are loaded, or have Constants defined here that
+are loaded into the Flask configuration.
+All modules and lookups get their configuration from the
+Flask config, rather than reading environment variables directly
+or by accessing this configuration directly.
+"""
+import os
+import random
+
+from dotenv import find_dotenv, load_dotenv
+
+
+# this will load all the envars from a .env file located in the project root (api)
+load_dotenv(find_dotenv())
+
+CONFIGURATION = {
+    'development': 'legal_api.config.DevConfig',
+    'testing': 'legal_api.config.TestConfig',
+    'production': 'legal_api.config.ProdConfig',
+    'default': 'legal_api.config.ProdConfig'
+}
+
+
+def get_named_config(config_name: str = 'production'):
+    """Return the configuration object based on the name.
+
+    :raise: KeyError: if an unknown configuration is requested
+    """
+    if config_name in ['production', 'staging', 'default']:
+        config = ProdConfig()
+    elif config_name == 'testing':
+        config = TestConfig()
+    elif config_name == 'development':
+        config = DevConfig()
+    else:
+        raise KeyError(f'Unknown configuration: {config_name}')
+    return config
+
+
+class _Config():  # pylint: disable=too-few-public-methods
+    """Base class configuration that should set reasonable defaults.
+
+    Used as the base for all the other configurations.
+    """
+
+    PROJECT_ROOT = os.path.abspath(os.path.dirname(__file__))
+
+    MSG_RETRY_NUM = int(os.getenv('MSG_RETRY_NUM', '5'))
+
+    LD_SDK_KEY = os.getenv('LD_SDK_KEY', None)
+
+    # urls
+    LEGAL_API_URL = os.getenv('LEGAL_API_URL', None)
+
+    # variables
+    LEGISLATIVE_TIMEZONE = os.getenv(
+        'LEGISLATIVE_TIMEZONE', 'America/Vancouver')
+
+    SQLALCHEMY_TRACK_MODIFICATIONS = False
+
+    # POSTGRESQL
+    DB_USER = os.getenv('DATABASE_USERNAME', '')
+    DB_PASSWORD = os.getenv('DATABASE_PASSWORD', '')
+    DB_NAME = os.getenv('DATABASE_NAME', '')
+    DB_HOST = os.getenv('DATABASE_HOST', '')
+    DB_PORT = os.getenv('DATABASE_PORT', '5432')
+    SQLALCHEMY_DATABASE_URI = 'postgresql://{user}:{password}@{host}:{port}/{name}'.format(
+        user=DB_USER,
+        password=DB_PASSWORD,
+        host=DB_HOST,
+        port=int(DB_PORT),
+        name=DB_NAME,
+    )
+
+    NATS_CONNECTION_OPTIONS = {
+        'servers': os.getenv('NATS_SERVERS', 'nats://127.0.0.1:4222').split(','),
+        'name': os.getenv('NATS_CLIENT_NAME', 'entity.filing.worker')
+
+    }
+    STAN_CONNECTION_OPTIONS = {
+        'cluster_id': os.getenv('NATS_CLUSTER_ID', 'test-cluster'),
+        'client_id': str(random.SystemRandom().getrandbits(0x58)),
+        'ping_interval': 1,
+        'ping_max_out': 5,
+    }
+
+    SUBSCRIPTION_OPTIONS = {
+        'subject': os.getenv('NATS_DIGITAL_CREDENTIALS_SUBJECT', 'error'),
+        'queue': os.getenv('NATS_QUEUE', 'error'),
+        'durable_name': os.getenv('NATS_QUEUE', 'error') + '_durable',
+    }
```

Comment:
This will be subscribing to the entity event subject so I think the subject line can be:

`'subject': os.getenv('NATS_ENTITY_EVENT_SUBJECT', 'entity.events'),`